flag/service/azure/template/template.go

```diff
@@ -0,0 +1,5 @@
+package template
+
+type Template struct {
+	Branch string
+}
```

Comment:
Adds flag to override the GitHub branch used for deploying templates. Defaults to master if not set.

Reply:
Maybe we should call it `Version` or something more generic, to hide that detail. My thinking here is if we decide to switch from Github to Azure storage or anything else, we probably don't want to change flag names.